src/main/java/net/fabricmc/loader/impl/game/GameProvider.java

```diff
@@ -38,6 +39,13 @@ public interface GameProvider { // name directly referenced in net.fabricmc.load
 	Path getLaunchDirectory();
 	boolean isObfuscated();
 	boolean requiresUrlClassLoader();
+	Set<BuiltinTransform> getBuiltinTransforms(String className);
+
+	enum BuiltinTransform {
```

Comment:
I'm unfamiliar with these terms. Specifically `STRIP_ENVIRONMENT` and `CLASS_TWEAKS`. I looked in FabricTransformer but the usage isn't immediately clear to me. Perhaps we could add some documentation?